autogpt_platform/backend/backend/util/settings.py

```diff
@@ -105,6 +105,33 @@ class Config(UpdateTrackingModel["Config"], BaseSettings):
         description="The port for agent server API to run on",
     )
 
+    backend_cors_allow_origins: List[str] = []
```

Comment:
```suggestion
    backend_cors_allow_origins: List[str] = Field(default_factory=list)
```
otherwise it becomes a shared object between all instances

Reply:
Also, `PydanticSettings` doesn't parse comma-separated lists into `list[str]` implicitly, so to make this work you'll have to add a [model validator](https://docs.pydantic.dev/2.9/concepts/validators/#model-validators) for this parsing step.

Reply:
It needs to be set as a list, like so:

`BACKEND_CORS_ALLOW_ORIGINS=['http://localhost:3000', 'http://127.0.0.1:3000']`